convert2rhel/toolopts.py

```diff
@@ -549,12 +514,6 @@ def _process_cli_options(self):
                 " configuration file. We're going to use the command line values."
             )
 
-        if (config_opts.activation_key or config_opts.password) and parsed_opts.password_from_file:
```

Comment:
Instead of removing this completely, should we change to check to use the config file instead of the password_from_file one?

Reply:
Don't we do this already? This was just for the warning when both were set

Reply:
I agree that this was just the warning, setting the values was done in a [different place](https://github.com/oamg/convert2rhel/pull/1058/files#diff-a7331917babf0a1504893963baab7ad842640877734e5dbeb39ebf97ec584565L411).